presto-main/src/main/java/com/facebook/presto/operator/scalar/UrlFunctions.java

```diff
@@ -129,6 +132,42 @@ public static Slice urlExtractParameter(@SqlType(StandardTypes.VARCHAR) Slice ur
         return null;
     }
 
+    @Nullable
+    @Description("encode url")
+    @ScalarFunction
+    @SqlType(StandardTypes.VARCHAR)
+    public static Slice urlEncode(@SqlType(StandardTypes.VARCHAR) Slice url)
+    {
+        if (url == null) {
```

Comment:
This is not needed. Presto will take care of returning null if any of the arguments to a function are null.